frontend/src/components/auth/Credentials.tsx

```diff
@@ -92,65 +99,83 @@ const Credentials = ({
     return (
-      <div style={{ textAlign: "center" }}>
-        <h1>Login</h1>
-        <form>
-          <div>
-            <input
-              type="email"
-              value={email}
-              onChange={(event) => setEmail(event.target.value)}
-              placeholder="username@domain.com"
-            />
-          </div>
-          <div>
-            <input
-              type="password"
-              value={password}
-              onChange={(event) => setPassword(event.target.value)}
-              placeholder="password"
-            />
-          </div>
-          <div>
-            <button
-              className="btn btn-primary"
-              type="button"
-              onClick={onLogInClick}
-            >
-              Log In
-            </button>
-          </div>
-          <GoogleLogin
-            clientId={process.env.REACT_APP_OAUTH_CLIENT_ID || ""}
-            buttonText="Login with Google"
-            onSuccess={(response: GoogleResponse): void => {
-              if ("tokenId" in response) {
-                onGoogleLoginSuccess(response.tokenId);
-              } else {
-                // eslint-disable-next-line no-alert
-                window.alert(response);
-              }
-            }}
-            onFailure={(error: GoogleErrorResponse) =>
-              // eslint-disable-next-line no-alert
-              window.alert(JSON.stringify(error))
-            }
-          />
-        </form>
-        <div>
-          <button
-            className="btn btn-primary"
-            type="button"
-            onClick={onSignUpClick}
+      <Flex h="100vh">
+        <Box w="47%">
+          <Flex
+            h="30%"
+            w="36%"
+            top="27%"
+            left="6%"
+            direction="column"
+            position="absolute"
+            justifyContent="space-between"
           >
-            Sign Up
-          </button>
-        </div>
-      </div>
-    );
+            <Box display="flex"  alignItems="flex-start">
+              <Text variant="login" position="absolute">
+                Log In
+              </Text>
+            </Box>
+            <Box>
+              <Input 
+                variant="login"
+                position="absolute"
+                placeholder="Your email address"
+                value={email}
+                onChange={(event) => setEmail(event.target.value)}
+              />
+            </Box>
+            <Box>
+              <Input 
+                variant="login"
+                type="password"
+                position="absolute"
+                placeholder="Your password"
+                value={password}
+                onChange={(event) => setPassword(event.target.value)}
+                />
+            </Box>
+            <Box>
+              <Button
+                variant="login"
+                position="absolute"
+                disabled={email === '' || password === ''}
+                _hover={
```

Comment:
So this is a super small nitpick but when the button is disabled and you hover over it, the cursor turns into a pointer, which kind of gives the illusion that it's clickable (even though it's not). You can slightly modify this to change that behaviour:
```suggestion
                _hover={
                  email && password
                    ? {
                        background: "teal.500",
                        transition:
                          "transition: background-color 0.5s ease !important",
                      }
                    : {}
                }
```

Essentially when we have both the username and password, the hover effect should occur, otherwise, there shouldn't be an effect at all.